front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -91,22 +92,46 @@ const CustomPageSettingsForm = ({
       is: (value: ProjectsFilterTypes) => value === 'topics',
       then: array().of(string()).min(1, formatMessage(messages.atLeastOneTag)),
     }),
-    area_id: string().when('projects_filter_type', {
-      is: (value: ProjectsFilterTypes) => value === 'areas',
-      then: string().required(formatMessage(messages.selectAnArea)),
-    }),
+    area_id: string()
+      .nullable()
+      .when('projects_filter_type', {
+        is: (value: ProjectsFilterTypes) => value === 'areas',
+        then: string()
+          .nullable()
```

Comment:
I've read the docs https://github.com/jquense/yup#schemanullable-schema, but I still can't figure out. Don't `nullable` contradict `required`?

Reply:
I don't fully understand but the validations seem to work better if we pass `null` as the default value. I think the logic is "it can be null, but to be valid, it needs to be a string"